config/routes.rb

```diff
@@ -30,6 +30,8 @@
     match 'hits/:host_id', to: 'hits#show', via: :get
 
     post ':organization_id/parameter', to: 'settings#set_org_parameter', constraints: { organization_id: %r{[^\/]+} }
+
+    match '/ui/*path', constraints: { path: /[^\?]+/ }, to: 'ui_requests#forward_request', via: :all
```

Comment:
Similar here, this is not an UI but an API request. Consider using `match /api/*path'`, to forward it to `https://localhost:24443/api/*path`.